styles/products.css

```diff
@@ -0,0 +1,64 @@
+.filters {
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  width: 20%;
+  padding: 1rem;
+  height: 100vh;
+  z-index: 10;
+}
+
+.filter-list {
+  border-right: 1px solid var(--primary-color);
+  height: 70vh;
+  overflow-y: scroll;
+}
+
+.filter-list::-webkit-scrollbar {
+  display: none;
+}
+
+.filter-heading {
+  font-size: 1.25rem;
+  margin-bottom: 0.75rem;
+  font-weight: var(--weight-semibold);
+}
+
+.filter-subheading {
+  font-size: 1rem;
+}
+
+.filter-item {
+  margin: 1.5rem 0;
+}
+
+.filter-item li {
+  margin: 0.5rem 0;
+}
+
+.filter-item #slider {
+  width: 80%;
+}
+
+.product-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(13.125rem, 1fr));
+  padding-left: 20%;
+  grid-gap: 0.5rem;
```

Comment:
Use only the gap proper. I think grid gap is not preferable now.

Reply:
thanks Mobashir...mage the changes